src/utilities/__tests__/astFromValue-test.js

```diff
@@ -33,10 +33,14 @@ describe('astFromValue', () => {
       { kind: 'BooleanValue', value: false }
     );
 
-    expect(astFromValue(null, GraphQLBoolean)).to.deep.equal(
+    expect(astFromValue(undefined, GraphQLBoolean)).to.deep.equal(
       null
     );
 
+    expect(astFromValue(null, GraphQLBoolean)).to.deep.equal(
```

Comment:
Ok, it makes sense. I added condition and test in 02d71a06b847d8f3e85bfb6836e2d75ab1425218